pkg/version.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+const (
+	// Name is the software name.
+	Name = "Hubble"
```

Comment:
name doesn't seem that necessary

Reply:
I guess I fear that ["Hublbe"](https://github.com/cilium/hubble/pull/162#issuecomment-592102864) could make a comeback. :)
I'm totally fine removing it though.